### ISSUE TITLE
Cannot run antq behind proxy

### ISSUE THREAD

**Issue by TimoKramer** [liquidz/antq#145]
Hi,
I wanted to run antq behind a proxy but it seems it is not able to make the http-calls in the read-pom-fn.

This is what I am seeing:
```
; eval (current-form): (antq options (fetch-deps options))
; (err) Execution error (ConnectException) at java.net.PlainSocketImpl/socketConnect (PlainSocketImpl.java:-2).
; (err) Connection timed out (Connection timed out)
```
In line 130 in maven.clj I think it should be `Connection timed out` instead of `Operation timed out` but I might be wrong.

Does antq obey to maven proxy-settings? I assume so but what could be wrong here? I can resolve my deps with `clj -P`...

**Comment by liquidz** [comment]
@TimoKramer Thanks for your reporting!
Oh, I I forgot about proxies.
I'll have a look.

**Comment by liquidz** [comment]
@TimoKramer Sorry for late reaction.
I supported to run antq behind proxy in dev branch.
antq uses proxy setting in `~/.m2/setting.xml` for now.

E.g.
```xml
<settings>
  <proxies>
    <proxy>
      <id>testproxy</id>
      <active>true</active>
      <protocol>https</protocol>
      <host>localhost</host>
      <port>3128</port>
      <username>foo</username>
      <password>bar</password>
    </proxy>
  </proxies>
</settings>
```

Could you try [dev](https://github.com/liquidz/antq/tree/dev) branch?

**Comment by liquidz** [comment]
@TimoKramer Just released v1.6.0 for now.
If you have any trouble, feel free to reopen :)
